src/System.Private.ServiceModel/tests/Scenarios/Security/TransportSecurity/Negotiate/NegotiateStream_Tcp_Tests.cs

```diff
@@ -0,0 +1,297 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Infrastructure.Common;
+using System;
+using System.Security.Cryptography.X509Certificates;
+using System.ServiceModel;
+using System.ServiceModel.Channels;
+using System.ServiceModel.Security;
+using Xunit;
+
+public static class NegotiateStream_Tcp_Tests
+{
```

Comment:
Question only: I recently added under the TransportSecurity tests a number of tests that use TransferMode.Streamed.  I did this because when I enabled NetTcp streaming, I noticed code coverage gaps in stream-specific areas in StreamedFramingRequestChannel. Could I ask you to look at code coverage for that class and decide whether there are still any gaps that might be filled if you added NegotiateStream tests that used TransferMode.Streamed? It can be a different PR if we see a need for it.  It looks like Buffered vs Streamed take different code paths that are related to transport security.